### ISSUE TITLE
new user group "Metadata Curators" (from "Curating mapsets metadata to be on par with today's standards")

### ISSUE THREAD

**Issue by h3oCharles** [ppy/osu-web#10108]
there's a discussion ( https://github.com/ppy/osu/discussions/22376 ) of adding a new user group (called Curators, or Metadata Curators) that would be responsible for curating and editing metadata of already existing ranked maps, as there are many inconsistencies within these before the subject was taken seriously in the ranking process.

TL;DR of that discussion:
- new user group "Metadata Curators" (further as MdCs)
- MdCs would look at all beatmapsets with leaderboards (for now) and would create the correct metadata for each of them
- some metadata can overlap between the same two songs, but not the same two beatmapsets
- add web-stored variables for Curated Metadata
- maybe add a toggle for showing curated metadata

current thoughts:
- given that NAT/GMT can now change tags on the web front of the beatmap database, the similar could be done with the rest of the metadata fields
- as Project Loved has a separate team for searching metadata, the same thought process could be applied to existing ranked maps
- hi Skubi
- when pitching this idea to BNs and a former BSC, they are skeptical, with reasons ranging from the current execution of the concept being a bandaid-solution, the scope being too elaborate, insufficient manpower for over 100k ranked mapsets, the task being "asinine", and [the booru-like tagging system](https://github.com/ppy/osu-web/issues/5852) being idealistic at best

feel free to read the discussion linked above for further context

**Comment by Skubers** [comment]
I'm here if any questions arise

**Comment by Ephemeralis** [comment]
Currently, in-depth alteration of beatmapset metadata requires what is known as a 'hax submit', aka a live (and potentially destructive) update. The tools on osu-web currently only adjust tags at a server level, meaning that any changes made don't apply to the in-client search on table.

As it stands, creating a usergroup to revise existing beatmap metadata would require someone (probably me) to make *thousands* of edits in this regard using a process that quite frankly, terrifies me. It plainly is not feasible without probably exhaustive development efforts to allow for serverside metadata alteration, and I expect that kind of thing to be firmly in the ballpark of a lazer-centric feature.

This is not to even begin mentioning the productivity black hole that metadata discussions often end up being. I have not seen a singular topic produce more ineffectual conversation in osu! to date - hinging an entire usergroup around it sounds like a recipe for disaster.

**Comment by Okorin** [comment]
Ephemeral's post was the more or less internal consensus. IMO if anything this should be some NAT/GMT level permission. ppy mentioned the eventual plan is for the web changes to be able to cascade down into the lazer client, but that's kind a far off and low priority (i guess, not his exact words and I don't claim to know what the priorities are)

Metadata discussions are a nightmare, yes, but I think getting a proactive group on it is an even worse idea than just leaving them be. It would probably stall any rule/guideline changes regarding metadata by a group of people having to identify which sets are affected, and then bemoaning that they'd need to change thousands of beatmaps.

Also the idea of a usergorup that becomes completely obsolete after metadata reaches a "ok everything's good now" status is kinda weird which is why I'd suggest adding this to existing usergroups

so to conclude, good idea, but proactively checking this stuff is probably not good for your mental health

**Comment by h3oCharles** [comment]
> Also the idea of a usergorup that becomes completely obsolete after metadata reaches a "ok everything's good now" status is kinda weird which is why I'd suggest adding this to existing usergroups

that group could look at pending maps, either by being requested by mappers or nominators, or looking at maps at their own accord

**Comment by cl8n** [comment]
generally supportive of this idea but it would need to wait on a lot of technical changes to BSS and metadata storage. the current online fields (tags, genre, language) don't have good support in the game client as-is, and updating maps as an admin would be even worse since metadata contributes to a beatmap's hash. I also feel like the whole "booru-like tags" thing should be figured out *before* this, to avoid potentially re-doing lots of work.

also I totally agree that metadata discussions can often be a "productivity black hole", but at the same time there already *are* a lot of people willing to engage with this stuff, and uninterested ppl don't have to participate. in that sense I think this proposal could actually be more productive than current, where we've got metadata specialists spread into sub-groups of BNs and the Loved team, a few others (like OP it seems) unable to directly contribute in some cases, and the rest of BNs expected to have working knowledge of metadata even though it's kinda tangential to mapping.

**Comment by h3oCharles** [comment]
> feel like the whole "booru-like tags" thing should be figured out _before_ this

should that be in a separate issue?
